### ISSUE TITLE
aria-label on svg, not tabindex container element

### ISSUE THREAD

**Issue by joshuakfarrar** [highcharts/highcharts#11442]
#### Expected behaviour

SSA's accessibility tool [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) reports that if `.highcharts-container` can be tabbed to, it must have an aria-label.

#### Actual behaviour

SSA's accessibility tool [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) is reporting that while `.highcharts-container` can be tabbed to, it does not have an aria-label. Other tools in use seem to pick up the child element's aria-label, that of `svg`, but ANDI does not.

I believe this lives around `js/modules/accessibility/components/ContainerComponent.js`

#### Affected browser(s)

All

**Comment by sebastianbochan** [comment]
Hi @sent1nel,
Thank you for reporting about the problem.

FYI @oysteinmoseng

**Comment by oysteinmoseng** [comment]
This can be configured in latest dev code, using the `lang.accessibility.graphicContainerLabel` option.
